src/Redux/api.tsx

```diff
@@ -655,13 +668,19 @@ const routes = {
   },
   sampleTestList: {
     path: "/api/v1/patient/{patientId}/test_sample/",
+    method: "GET",
+    TRes: Type<PaginatedResponse<SampleTestModel>>(),
   },
   createSampleTest: {
     path: "/api/v1/patient/{patientId}/test_sample/",
     method: "POST",
+    TRes: Type<PatientModel>(),
+    TBody: Type<SampleTestModel>(),
   },
   sampleReport: {
     path: "/api/v1/patient/{id}/test_sample/{sampleId}/icmr_sample",
```

Comment:
Missing trailing slash. Ensure trailing slash is present for all APIs

```suggestion
    path: "/api/v1/patient/{id}/test_sample/{sampleId}/icmr_sample/",
```